types.d.ts

```diff
@@ -17,8 +17,10 @@ declare module 'discord-rose/dist/typings/lib' {
 type msgFunction = (ctx: CommandContext) => string
 
 declare const _default: (msgs?: {
```

Comment:
This would also no longer be categorized as `msgs`
```suggestion
declare const _default: (opts?: {
```